sinks/cache/gc_store.go

```diff
@@ -43,13 +44,15 @@ func (gcs *gcStore) Delete(start, end time.Time) error {
 
 func (gcs *gcStore) reapOldData(timestamp time.Time) {
 	end := timestamp.Add(-gcs.bufferDuration)
-	start := time.Time{}
-	if err := gcs.store.Delete(start, end); err != nil {
-		glog.Fatalf("failed to delete old data - %v", err)
+	if end.Before(time.Unix(0, 0)) {
```

Comment:
I don't think I intended to make this change. Merge issue maybe? Will fix it anyways.